lib/text_diff.ml

```diff
@@ -40,6 +43,20 @@ let lbl_to_lines ld =
   Format.pp_print_flush formatter ();
   CCString.lines (Buffer.contents buf)
 
+let te_to_lines te =
```

Comment:
what does `te` stands for here? Is it tuple?

Reply:
It stands for type_expr. Sorry but I followed the same naming convention as in the other to_lines functions :)